generate-feedback/main.py

```diff
@@ -0,0 +1,175 @@
+        ]
+        choice = random.choice(options)
+        return choice["print"], choice["raw"]
+
+    return "", ""
+
+
+def generate_feedback_data(feedback_id, course_name, teacher_name, num_students):
+    # Construieste structura JSON pentru un singur formular
+    anon_attempts = []
+
+    base_attempt_id = feedback_id
+    base_response_id = feedback_id * 200
+
+    current_response_global_counter = base_response_id
+
+    questions_structure = [
+        ("Subject", "fixed", course_name),
+        ("Teacher", "fixed", teacher_name),
+        ("Laboratory/seminar/project ...", "fixed", teacher_name),
+        ("Is your assessment of this ...", "likert", ""),
+        ("What grade do you expect to...", "grade", ""),
+        ("Is the general workload in ...", "likert", ""),
+        ("Location / hardware and ...", "likert", ""),
+        ("The approximate number of ...", "percent", ""),
+        ("Does the course tutor have ...", "likert", ""),
+        ("Was the teaching method ...", "likert", ""),
+        ("Did the course stimulate ...", "likert", ""),
+        ("Other personal comments or ...", "text", "")
+    ]
+
+    for i in range(num_students):
+        responses = []
+        current_attempt_id = base_attempt_id + i
+
+        # Sablonul pt afisarea datelor
+        for q_name, q_type, q_default in questions_structure:
+            entry = {
+                "id": current_response_global_counter,
+                "name": q_name,
+                "printval": "",
+                "rawval": ""
+            }
+
+            if q_type == "fixed":
+                entry["printval"] = q_default
+                entry["rawval"] = q_default
+            elif q_type == "text":
+                val = "feedback scris" if random.random() > 0.8 else ""
+                entry["printval"] = val
+                entry["rawval"] = val
+            else:
+                p_val, r_val = get_random_response(q_type)
+                entry["printval"] = p_val
+                entry["rawval"] = r_val
+
+            responses.append(entry)
+            current_response_global_counter += 1
+
+        anon_attempts.append({
+            "id": current_attempt_id,
+            "courseid": 0,
+            "number": i + 1,
+            "responses": responses
+        })
+
+    return {
+        "attempts": [],
+        "totalattempts": 0,
+        "anonattempts": anon_attempts,
+        "totalanonattempts": len(anon_attempts),
+        "warnings": []
+    }
+
+
+def main():
+    feedbacks = load_pickle('feedbacks.p')
+    courses = load_pickle('courses.p')
+    categories = load_pickle('categories.p')
+
+    courses_map = {c['id']: c for c in courses}
+    categories_map = {cat['id']: cat for cat in categories}
+
+    output_dir = 'feedback_contents'
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+        print(f"Folder created: '{output_dir}'")
+
+    print(f"Loaded {len(feedbacks)} feedback forms and {len(courses)} courses.")
```

Comment:
Do not use print to log messages, use [Python logging facility](https://docs.python-guide.org/writing/logging/). Take a bit of time to look into this, and then use logging. Configure it according to your needs and to the project specifics.